bin/Project.rei

```diff
@@ -66,6 +66,8 @@ let buildDependencies:
   ) =>
   RunAsync.t(unit);
 
+let scanDependencies: (project, BuildSandbox.Plan.t) => RunAsync.t(PathSet.t);
```

Comment:
Instead of adding `scanDependencies` I'd inline the code in `esy gc` command source code and add `Project.renderSandboxPath(Project.t, SandboxPath.t)` utility.